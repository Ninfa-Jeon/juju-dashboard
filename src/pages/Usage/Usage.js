import React from "react";
import Layout from "components/Layout/Layout";

export default function Usage() {
  return (
    <Layout>
      <div className="p-strip">
        <div className="row">
          <h2>Usage</h2>
        </div>
      </div>
    </Layout>
  );
}
