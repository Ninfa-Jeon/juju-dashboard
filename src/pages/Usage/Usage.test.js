import React from "react";
import { shallow } from "enzyme";
import Usage from "./Usage";

describe("Usage page", () => {
  it("renders without crashing and matches snapshot", () => {
    const wrapper = shallow(<Usage />);
    expect(wrapper).toMatchSnapshot();
  });
});
