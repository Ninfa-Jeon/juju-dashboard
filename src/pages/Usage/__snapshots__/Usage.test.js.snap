// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Usage page renders without crashing and matches snapshot 1`] = `
<Layout>
  <div
    className="row"
  >
    <Notification
      type="caution"
    >
      We're still working on this functionality - please check back soon!
    </Notification>
  </div>
</Layout>
`;
